README.md

```diff
@@ -1,3 +1,12 @@
+<p align="center"><img src="https://www.empirical-soft.com/logo.png"></p>
+
+<p align="center">
+	<a href="https://www.gnu.org/licenses/agpl-3.0.en.html"><img src="https://img.shields.io/badge/License-AGPL%20v3-blue.svg" alt="Affero General Public License (AGPL) with the Commons Clause"></a>
```

Comment:
Thanks for taking a crack at this. I'd remove the license shield since I don't want anyone to think that it's just AGPL; enforcing the Commons Clause is a very specific part of my business plan.